packages/app/src/editors/markdown/credits.md

```
@@ -0,0 +1,12 @@
+In developing the Markdown editor, we have referred to and reused some code from the following projects.
+
+We would like to express our gratitude to the project authors for their excellent work.
+
+----
+
+**Project:** ProseMark  
+**URL:** https://github.com/jsimonrichard/ProseMark/  
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)